curriculumBuilder/settings.py

```diff
@@ -545,11 +546,11 @@ def iframe_filter_domain(name, value):
 AWS_BASE_URL = 'http://cdo-curriculum.s3-website-us-east-1.amazonaws.com'
 
 STATICFILES_LOCATION = 'static'
-STATICFILES_STORAGE = 'curriculumBuilder.s3utils.StaticRootS3BotoStorage'
+STATICFILES_STORAGE = 'helpers.s3utils.StaticRootS3BotoStorage'
```

Comment:
Why the file move? not opposed to it, just wondering.

Reply:
Arguably arbitrary personal preference. I thought it'd make sense to put pdfUtils into a 'helpers' module since it doesn't really seem to fit in anywhere. While I was at it, I figured I'd put the s3utils module in there too, since it also seems like a 'helper' and it bothered me that there was a file with the suffix utils in a module that isn't a utils module. 

I don't feel suuuper strongly about it being here vs somewhere else.